client.go

```diff
@@ -69,6 +80,13 @@ func (c *Client) doRequest(method string, path string, body io.Reader, v interfa
 		return err
 	}
 
+	// print body for dev
```

Comment:
This code block is probably just yours for debug. Can you remove this please?

Reply:
Sure. I've also extended events as well to get the list of attendees, organizer and a couple of other details, so I need to amend this PR anyway. I did not however include create methods for all those values- I didn't want to mess up your methods with a dozen new properties. Is that OK?

Reply:
Ok, done. I was able to ditch the new create method, and slim down the code too. Went with the get token method only- not detect change. I can't think of why the library would care about a refresh- it's already done it. AS long as the calling app has visibility to the updated token, all is fine.  @jkrecek please merge when you have time.